harbor-ui/src/main/content/jcr_root/apps/harbor/components/content/text/v1/text/README.md

```diff
@@ -0,0 +1,19 @@
+# Harbor Text Component
+
+A simple text component for editing and displaying text. Uses an in-place Rich Text Editor for authoring.
+
+## General Information
+
+* `group`: Harbor
+* `resourceType`: `harbor/components/content/text/v1/text`
+* `resourceSuperType`: none
+
+## Sling Model
+
+Uses the `com.icfolson.aem.harbor.api.components.content.text.Text` Model interface.
+
+## Authorable Properties
+
+* `content`: The text to present in the Text component.
```

Comment:
we need to add some indication that this is using the text inline editor for authoring.  Maybe break this section into two pieces - one with the dialog fields and one with the inline editors used.